packages/flutter_tools/templates/create/lib/main.dart.tmpl

```diff
@@ -55,6 +58,19 @@ class MyHomePage extends StatefulWidget {
 
 class _MyHomePageState extends State<MyHomePage> {
   int _counter = 0;
+{{#withPluginHook}}
+  String _platformVersion = 'Unknown';
+
+  @override
+  void initState() {
+    super.initState();
+    {{pluginDartClass}}.platformVersion.then((String platformVersion) {
+      setState(() {
+        _platformVersion = platformVersion;
```

Comment:
There is no way to cancel. Added mounted check in https://github.com/flutter/flutter/issues/9458.